### DISCUSSION TITLE
首页 Logo 旁边那个 MOD 存在的必要性是什么？我想去掉它

### DISCUSSION THREAD

**Discussion by ShenQingchuan** [usememos/memos#2676] [Q&A]
看到源码里是这样定义的：
https://github.sheincorp.cn/usememos/memos/blob/cc40803b065f6d913dcd5d0e6158df0efc6fdfed/web/src/components/UserBanner.tsx#L42-L44

想问下这个的意义是什么？我觉得挂在那里很奇怪，能否不显示？

**Comment by boojack** [comment]
It will be removed in `v0.18.2`.
